src/i18n/strings/en_EN.json

```diff
@@ -2182,8 +2182,6 @@
     "Room Notification": "Room Notification",
     "Notification Autocomplete": "Notification Autocomplete",
     "Room Autocomplete": "Room Autocomplete",
-    "Users": "Users",
```

Comment:
I ran `gen-i18n`... not sure why it is remove `"Users"` though :thinking:

Reply:
> src\autocomplete\UserProvider.tsx (0 strings)

... whaaat

Reply:
so this is because our "flow is probably fine" mentality is starting to fail us: https://github.com/matrix-org/matrix-react-sdk/pull/5077